abs.yaml

```diff
@@ -0,0 +1,5 @@
+aggregate_branch: 3.12
+
+build_parameters:
```

Comment:
Any specific reason for overriding the default build parameters?

Reply:
yes,  try to run skipped window build but we have a bug on CI, see
https://anaconda.slack.com/archives/C03J3NQPS0Z/p1707163429250309